packages/react-native-sdk/src/components/CallParticipantsInfoView.tsx

```diff
@@ -110,38 +125,99 @@ const InnerCallParticipantsInfoView = () => {
     }
   };
 
-  return (
-    <>
-      <View style={styles.buttonGroup}>
-        <Restricted requiredGrants={[OwnCapability.MUTE_USERS]}>
-          <Pressable style={styles.button} onPress={muteAllParticipantsHandler}>
-            <Text style={styles.buttonText}>Mute All</Text>
-          </Pressable>
-        </Restricted>
-      </View>
-      <FlatList
-        data={participants}
-        keyExtractor={(item) => `participant-info-${item.sessionId}`}
-        renderItem={({ item: participant }) => (
-          <CallParticipantInfoItem
-            participant={participant}
-            setSelectedParticipant={setSelectedParticipant}
-          />
-        )}
+  const onCloseCallParticipantsViewVisible = () => {
+    setIsCallParticipantsViewVisible(false);
+  };
+
+  const renderItem = useCallback(({ item }: { item: Participant }) => {
+    return (
+      <CallParticipantInfoItem
+        participant={item}
```

Comment:
also I would add key here and remove the keyextractor.. so none of the dependencies except data to flatlist will be changing on every render then